File: HiveHelper/Models/ActionDetail.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;

namespace HiveHelper.Models
{
    public class ActionDetail
    {
        public long id { get; set; }
        public long primary_action_id { get; set; }
        public long secondary_action_id { get; set; }
        public long tertiary_action_id { get; set; }
        public long hive_id { get; set; }
        public long completed_by_id { get; set; }
        public long entered_by_id { get; set; }
        public bool completed { get; set; }
        public DateTime entry_date { get; set; }
        public DateTime completed_date { get; set; }
        public DateTime scheduled_date { get; set; }
        public string comments { get; set; }
        public string primary_action_name { get; set; }
        public string secondary_action_name { get; set; }
        public string tertiary_action_name { get; set; }
        public string completed_by_first_name { get; set; }
        public string completed_by_last_name { get; set; }
        public string entered_by_first_name { get; set; }
        public string entered_by_last_name { get; set; }
    }
}

File: HiveHelper/ClientApp/src/app/models/action-detail.ts
export interface ActionDetail {
  id: number;
  primary_action_id: number;
  secondary_action_id: number;
  tertiary_action_id: number;
  hive_id: number;
  completed_by_id: number;
  entered_by_id: number;
  completed: boolean;
  entry_date: Date | string;
  completed_date: Date | string;
  scheduled_date: Date | string;
  comments: string;
  primary_action_name: string;
  secondary_action_name: string;
  tertiary_action_name: string;
  completed_by_first_name: string;
  completed_by_last_name: string;
  entered_by_first_name: string;
  entered_by_last_name: string;
}
